src/test/java/uk/gov/hmcts/reform/sscs/service/ServiceHearingsServiceTest.java

```diff
@@ -278,7 +261,174 @@ void testGetServiceLinkedCaseReturnsLinkedCases() throws Exception {
+                .wantsToAttend("Yes")
+                .excludeDates(
+                    List.of(
+                        ExcludeDate.builder().value(DateRange.builder().end("2023-01-01").build()).build(),
+                        ExcludeDate.builder().value(DateRange.builder().start("").end("2023-01-01").build()).build(),
+                        ExcludeDate.builder().value(DateRange.builder().start(null).end("2023-01-01").build()).build()
+                    )
+                )
+                .build()
+        );
+
+        given(ccdCaseService.getCaseDetails(String.valueOf(CASE_ID))).willReturn(caseDetails);
+        ServiceHearingRequest request = ServiceHearingRequest.builder()
+            .caseId(String.valueOf(CASE_ID))
+            .build();
+        ServiceHearingValues serviceHearingValues = serviceHearingsService.getServiceHearingValues(request);
+        serviceHearingValues.getParties().stream()
+            .map(PartyDetails::getUnavailabilityRanges)
+            .forEach(o -> o.forEach(p -> {
+                assertThat(p.getUnavailableFromDate()).isEqualTo(p.getUnavailableToDate());
+                assertThat(p.getUnavailabilityType()).isEqualTo(ALL_DAY.getLabel());
+            }));
+        verify(ccdCaseService, times(1)).updateCaseData(any(SscsCaseData.class), eq(UPDATE_CASE_ONLY), anyString(), eq("IDs updated for caseDetails due to ServiceHearingValues request"));
+    }
+
+    @Test
+    void testGetServiceHearingValues_PartiesUnavailabilityEndDateIsBeforeStartDate_ThenSentItToListingError() throws Exception {
+        given(sessionCategoryMaps.getSessionCategory(BENEFIT_CODE, ISSUE_CODE, true, false))
+            .willReturn(new SessionCategoryMap(BenefitCode.PIP_NEW_CLAIM, Issue.DD, false,
+                                               false, SessionCategory.CATEGORY_03, null));
+        given(referenceDataServiceHolder.getSessionCategoryMaps()).willReturn(sessionCategoryMaps);
+        given(venueService.getEpimsIdForVenue(caseData.getProcessingVenue())).willReturn("9876");
+        given(referenceDataServiceHolder.getVenueService()).willReturn(venueService);
+
+        caseDetails.getData().getAppeal().setHearingOptions(
+            HearingOptions.builder()
+                .wantsToAttend("Yes")
+                .excludeDates(
+                    List.of(
+                        ExcludeDate.builder().value(DateRange.builder().start("2023-01-01").end("2023-01-02").build()).build(),
+                        ExcludeDate.builder().value(DateRange.builder().start("2023-02-01").end("2023-02-02").build()).build(),
+                        ExcludeDate.builder().value(DateRange.builder().start("2023-03-01").end("2023-03-02").build()).build(),
+                        ExcludeDate.builder().value(DateRange.builder().start("2023-05-01").end("2023-04-02").build()).build(),
+                        ExcludeDate.builder().value(DateRange.builder().start("2023-07-01").end("2023-08-02").build()).build()
```

Comment:
This suggestion might be overkill (bc how many dates you are using) but I'll mention it anyway. It is tricky to see what you are trying to achieve with these dates. I can read the name of the test, but most of the pairs are fine, and I also had to use some amount of brain power to process what they all are anyway. It would be really nice (from a reviewer pov) if you saved your dates as variables something like date1, date1After, date2, date2Before etc. This means I don't even have to think, I can just see at a glance what you are testing and where the anomaly is.
It's up to you though - something to have in your arsenal.